main.tf

```diff
@@ -0,0 +1,32 @@
+resource "google_storage_bucket" "auto-expire" {
+  name          = var.name
+  project       = var.project
+  location      = "US"
```

Comment:
Could we place that as default variables or pull from somewhere where it might be reused